deployment/molecule/repo/tests/test_non_existing.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.mark.parametrize("path", [
+    ("/home/poadocker/bridge/node_modules"),
```

Comment:
What should be done as part of the new approach in order to this directories will be copied? I mean, if someone changes the behavior intentionally to copy these directories why will he/she stop to not remove these checks?

Reply:
If someone intentionally wants to copy these directories then he will remove the checks.

I have in mind unintentional copying, when configuration on what to ignore when copying is unintentionally broken.

Reply:
For example right now I am fighting with copying configuration to ignore `.git` when copying, but it doesn't work for me for some reason. 
Despite setting `exclude`, the `.git` gets copied to remote.
https://github.com/poanetwork/tokenbridge/blob/21480f25603853d27abd8572116332e36f88aa81/deployment/roles/common/tasks/repo.yml#L11
If I get it to work, still I think it would be good if it was covered by tests.

Reply:
I see now